remill/Arch/AArch64/Semantics/SIMD.cpp

```diff
@@ -855,6 +855,34 @@ DEF_SEM(NOT_8, V128W dst, S src) {
 DEF_ISEL(NOT_ASIMDMISC_R_8B) = NOT_8<V64>; 
 DEF_ISEL(NOT_ASIMDMISC_R_16B) = NOT_8<V128>; 
 
+namespace {
+
+#define MAKE_EXT(size, count) \
```

Comment:
Here I'd recommend using a template rather than a macro, and to get the `count`, where you could have a source type `V64` in one case, and `V128` in another, and you can also pass in a `size_t count` in the template parameter list, or infer `count` using `NumVectorElements(vn)`.